### ISSUE TITLE
Configurable background color

### ISSUE THREAD

**Issue by r-darwish** [PowerShell/GraphicalTools#129]
The default colors don't look well on every theme. It would be nice if we could configure them.

**Comment by tig** [comment]
Can you share more details of what looks good and doesn't on your system?

`Terminal.Gui`, on which `ocgv` is based supports multiple color themes. One solution would be to add a switch to `ocgv` that lets the user choose these:

Base:
![image](https://user-images.githubusercontent.com/585482/105528716-2cb96a80-5c9a-11eb-824f-ba1c1075f4de.png)

TopLevel:
![image](https://user-images.githubusercontent.com/585482/105528744-3642d280-5c9a-11eb-832d-bc499b663381.png)

Dialog:
![image](https://user-images.githubusercontent.com/585482/105528790-435fc180-5c9a-11eb-901b-f270ba08270a.png)

Error:
![image](https://user-images.githubusercontent.com/585482/105528815-4ce92980-5c9a-11eb-8dc7-f2a22b2e2fb5.png)

Finding color schemes that work well cross-platform is a real challenge. We do have the ability to tweak the defaults across some platforms so if there's something in one of the defaults that looks awful on, say Linux, we have some control.

**Comment by freMea** [comment]
Yes maybe a general theme setting for all terminal.gui applications or/and a specific command to set ocgv theme like `set-ocgvColors` and passing a hash table with colors like we do with `Set-PSReadLineOption -Colors $myTheme`

**Comment by freMea** [comment]
Yes

**Comment by tig** [comment]
Ok. Thank you for the feedback!

For 1. see https://github.com/migueldeicaza/gui.cs/issues/1773

For 2. see https://github.com/migueldeicaza/gui.cs/issues/29

**Comment by tig** [comment]
https://github.com/gui-cs/Terminal.Gui/issues/1773 has been fixed and will be in Terminal.Gui v1.7.2 which I plan on launching in the next day. It will be incorporated into the next `GraphicalTools` release.

<img width="327" alt="image" src="https://user-images.githubusercontent.com/585482/182746445-6dff8805-db39-4878-8c22-7a32accebebb.png">

It's not a dramatic change, but does make the not-focused but selected listview item easier to read.

Providing customizable colors is a future item...

**Comment by tig** [comment]
@andschwa please prefix title with "OCGV: ".

**Comment by tig** [comment]
Terminal.Gui v2.x will fix this. 

See https://github.com/gui-cs/Terminal.Gui/issues/2181

I plan on a re-write of OCGV using Terminal.Gui v2 in the next few months.

**Comment by HotCakeX** [comment]
Really need a way to customize background color...
